modules/component-store/tsconfig.build.json

```diff
@@ -22,11 +22,11 @@
   "files": ["public_api.ts"],
   "angularCompilerOptions": {
     "annotateForClosureCompiler": true,
+    "compilationMode": "partial",
```

Comment:
I'm a bit out of sync with the v13 release.
But if I understand correctly, removing the `enableIvy` option and setting `compilationMode` to `partial` makes a library ivy ready?

Reply:
Yep, enableIvy is true by default and compilationMode should be partial in v13